src/app/shared/example-viewer/example-viewer.html

```diff
@@ -2,7 +2,16 @@
   <div class="docs-example-viewer-title" *ngIf="view !== 'snippet'">
     <div class="docs-example-viewer-title-spacer">{{exampleData?.title}}</div>
 
-    <button mat-icon-button type="button" (click)="toggleCompactView()" [matTooltip]="'View snippet only'"
+    <button
+      mat-icon-button
+      type="button"
+      aria-label="Copy link to this example to clipboard"
```

Comment:
Is there enough context here for a screen reader viewer to understand what "this" means? Maybe not if they are navigating by button landmarks.
We probably should include the example description here like the stack-blitz-button does.